todo_app/app.py

```diff
@@ -1,47 +1,92 @@
 import os
+import logging
+import logging.config
+import loggly.handlers
+import time
+from dotenv import load_dotenv
 from flask import Flask, render_template, request, redirect, url_for
 from todo_app.flask_config import Config
 from todo_app.data.view_model import ViewModel
 from todo_app.data.items import ItemService
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+
+logging.config.fileConfig('python.conf')
```

Comment:
Is there a reason you've added the file config as well as the defined logging below? It feels like duplication